### PR TITLE
BUG FIX initPlayerLocal.sqf

### PR DESCRIPTION
**Pull request by AlexTriada**
Fix wrong "InitializeGroup" parameter for function BIS_fnc_dynamicGroups

### CODE REVIEW

File: A3-Antistasi/initPlayerLocal.sqf [hunk 1]
```diff
@@ -432,26 +427,22 @@ player addEventHandler ["GetInMan",
 	}
 	];
 
-if (isMultiplayer) then
-	{
+if (isMultiplayer) then {
 	["InitializePlayer", [player]] call BIS_fnc_dynamicGroups;//Exec on client
-	["InitializeGroup", [player,teamPlayer,true]] call BIS_fnc_dynamicGroups;
```
Comment (AlexTriada) [inline A3-Antistasi/initPlayerLocal.sqf]: that fix this:

> 2:19:29 "triada/log: ERROR: [BIS_fnc_dynamicGroups] Unknown mode: InitializeGroup"

File: A3-Antistasi/initPlayerLocal.sqf [hunk 2]
```diff
@@ -432,26 +427,22 @@ player addEventHandler ["GetInMan",
...
+
```
Comment (AlexTriada) [inline A3-Antistasi/initPlayerLocal.sqf]: that fix this:

> 2:19:29 "triada/log: ERROR: [BIS_fnc_dynamicGroups] Unknown mode: InitializeGroup"

File: A3-Antistasi/initPlayerLocal.sqf [hunk 3]
```diff
@@ -48,7 +48,7 @@ if (isMultiplayer) then
 	tkPunish = if ("tkPunish" call BIS_fnc_getParamValue == 1) then {true} else {false};
 	if ((side player == teamPlayer) and tkPunish) then
 	{
-		private _firedHandlerTk =
```
Comment (AlexTriada) [inline A3-Antistasi/initPlayerLocal.sqf]: it's automatic delete space in the end of the strings

**Comment by Spoffy** [comment]
I'll review and merge this and your other PRs after the next release. I'm trying to minimise changes that aren't bugfixes right now, to keep the number of things we break to a minimum.

**Comment by AlexTriada** [comment]
it's a bugFIX, affecting to load speed

**Pull request closed without merging**